packages/doenetml-worker-rust/lib-doenetml-core/src/core/components/common.rs

```diff
@@ -8,7 +8,7 @@ use serde::{Deserialize, Serialize};
 
 use crate::dast::{DastAttribute, Position as DastPosition};
 use crate::state::{
-    StateVar, StateVarName, StateVarReadOnlyView, StateVarReadOnlyViewTyped, StateVarValue,
+    StateVarEnum, StateVarName, StateVarReadOnlyView, StateVarReadOnlyViewEnum, StateVarValueEnum,
```

Comment:
While we're renaming things...Is there a different type of view for a state var? Can we call it `StateVarView` instead of `StateVarReadOnlyView`?

Reply:
We have both a `StateVarMutableView` and a `StateVarReadOnlyView`.

The `StateVarMutableView` is used, for example, in `calculate_state_var_from_dependencies_and_mark_fresh` because in that function, it sets the value of the state variable.